### PR TITLE
LPS-113577 Deprecates `liferay-message` module

### PR DESCRIPTION
**Pull request by diegonvs**
Since this module isn't being used on Document Library, I just removed the unnecessary imports of this component from document-library-web and placed a notice about the deprecation on the module.

Established that there are no current usages by grepping:

- [`git grep '\bLiferay\.Message\b'`](https://gist.github.com/wincent/94efccb73bb591c71f0770b6409a67ad)
- [`git grep liferay-message`](https://gist.github.com/wincent/cf6db876eadbdce3f670ecba10a0ab9b)

Looks like last actual usages were removed in 54c332cdef1a8ba16cc1aa3ae0266ef6dc872bd2 ("LPS-52907 Remove Liferay sync message from Documents and Media", Jan 2015) and 82a686413063c7650c1270f7b765f48fa4549340 ("LPS-35401 Clean up js code which doesn't seem to be needed any more", May 2013).

(Found via `git log -GLiferay\.Message -p`).

/cc @jbalsas @wincent

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
Screen Shot 2020-05-21 at 09 49 31

**Comment by wincent** [comment]
@diegonvs: lg2m. Before forwarding, we should edit the description with information about how you verified that it is unused. I'd expect to see something like:

---

Established that there are no current usages by grepping:

- [`git grep '\bLiferay\.Message\b'`](https://gist.github.com/wincent/94efccb73bb591c71f0770b6409a67ad)
- [`git grep liferay-message`](https://gist.github.com/wincent/cf6db876eadbdce3f670ecba10a0ab9b)

Looks like last actual usages were removed in 54c332cdef1a8ba16cc1aa3ae0266ef6dc872bd2 ("LPS-52907 Remove Liferay sync message from Documents and Media", Jan 2015) and 82a686413063c7650c1270f7b765f48fa4549340 ("LPS-35401 Clean up js code which doesn't seem to be needed any more", May 2013).

(Found via `git log -GLiferay\.Message -p`).

---

Bonus for linking to a gist containing the output.

Basically, the idea is that you've already done the hard work of finding out what change to make — but then you can make it easy for the reviewer to have confidence in your change and confirm your analysis without having to actually redo it themselves. (ie. in this case, I had to do the analysis to produce the above description, but I am lazy and would much prefer it if I didn't have to... I guess I could have just trusted but we supposedly review because we believe in verification, not just trust 😂)

I edited the description for you but can you please also edit the title of the PR so that it shows up in Brian's queue as "LPS-113577 Deprecates `liferay-message` module" instead of just "LPS 113577"? Thanks.

**Comment by wincent** [comment]
> I edited the description for you but can you please also edit the title of the PR so that it shows up in Brian's queue as "LPS-113577 Deprecates liferay-message module" instead of just "LPS 113577"? Thanks.

I lie. I didn't edit it and I can't edit it. Can you please copy paste:

```
Established that there are no current usages by grepping:

- [`git grep '\bLiferay\.Message\b'`](https://gist.github.com/wincent/94efccb73bb591c71f0770b6409a67ad)
- [`git grep liferay-message`](https://gist.github.com/wincent/cf6db876eadbdce3f670ecba10a0ab9b)

Looks like last actual usages were removed in 54c332cdef1a8ba16cc1aa3ae0266ef6dc872bd2 ("LPS-52907 Remove Liferay sync message from Documents and Media", Jan 2015) and 82a686413063c7650c1270f7b765f48fa4549340 ("LPS-35401 Clean up js code which doesn't seem to be needed any more", May 2013).

(Found via `git log -GLiferay\.Message -p`).
```

**Comment by diegonvs** [comment]
I gonna rebase then thanks for the heads-up Greg

**Comment by diegonvs** [comment]
Also I gonna edit the commit message

**Comment by diegonvs** [comment]
ci:forward

**Comment by wincent** [comment]
Thanks @diegonvs!

**Comment by wincent** [comment]
Asked about current bot status [in Slack](https://liferay.slack.com/archives/C19PWQ74J/p1590074948371700).
